sc/src/main/java/uk/ac/ebi/atlas/experimentpage/CellMetadataService.java

```diff
@@ -59,7 +60,7 @@ public CellMetadataService(IdfParser idfParser, SolrCloudCollectionProxyFactory
 
         SingleCellAnalyticsSchemaField[] attributeFields = idfParserOutput.getMetadataFieldsOfInterest()
                 .stream()
-                .map(attribute -> SingleCellAnalyticsCollectionProxy.characteristicAsSchemaField(attributeNameToFieldName((attribute))))
+                .map(attribute -> SingleCellAnalyticsCollectionProxy.characteristicAsSchemaField(attributeNameToFieldName((attribute)), attribute))
```

Comment:
We all like Lisp, but those double parenthesis could be removed. 😜

Reply:
`(print "oops!")`